mlrun/serving/routers.py

```diff
@@ -611,7 +648,7 @@ def _mean_vote(self, all_predictions: List[List[float]], weights: List[float]):
         return (np.array(all_predictions) @ weights).tolist()
 
     def _is_int(self, value):
-        return float(value).is_integer()
+        return value % 1 == 0
```

Comment:
I meant replacing the `_is_int` entirely as its not shortening anything. See if `_is_int` is used and where its used just replace it with `% 1 == 0`. Better we leave it for the next PR where we rearrange the code here a bit. So for now, rollback this change to the original.